data_steward/deid/config/ids/tables/condition_occurrence.json

```diff
@@ -1,8 +1,20 @@
+        },
```

Comment:
should these queries be `condition_occurrence_id = record2.condition_occurrence_id` instead of `condition_occurrence.condition_occurrence_id = record2.condition_occurrence_id`? When testing these queries in BQ it is getting an error when looking for the `condition_occurrence.` alias.

Reply:
I didn't want to change these because they were working before, but you're right, they don't make sense as-is.